### PR TITLE
feat: Cards Layout

### PR DESCRIPTION
**Pull request by tushartiwari7**
Added all layouts for cards component

- Text Only
- Horizontal
- dismissable
- card with badge
- Text over Display

### CODE REVIEW

**Review by krishnakant01 — APPROVED** [review]
code is really clean and easy to read! cool use of fetch(), you can merge it.

File: components/card/textOnly/preview.html [hunk 1]
```diff
@@ -1,18 +1,11 @@
-<h4 class="title fs-l">Text Only Card Button</h4>
+<h4 class="title fs-l">Text Only Card </h4>
 <div class="btn-wrapper preview-container rounded-s p-mid flex flex-center fullwidth">  
   <section class="card pos-rel">
-    <!-- <div class="flex flex-col pos-rel text-over-media">
-      <img class="card-img" src="https://placeimg.com/200/100" alt="card image">
-      
-    </div> -->
```
Comment (Bhakti199) [inline components/card/textOnly/preview.html]: remove extra comments

**Review by Bhakti199 — COMMENTED** [review]
overall code is nicely written and structured.

File: components/card/textOnly/preview.html (hunk 1 above)
Comment (tushartiwari7) [reply components/card/textOnly/preview.html]: It was already resolved in the latest [commit](https://github.com/tushartiwari7/elevate-ui/pull/13/commits/67ee2f8af1be75c7456cfdfa1c8b9b34c8b22588). please have a look again @Bhakti199

**Review by Bhakti199 — APPROVED** [review]

**Pull request merged**